packages/react-instantsearch-dom/src/core/createInstantSearchServer.js

```diff
@@ -60,7 +62,10 @@ const createInstantSearchServer = algoliasearch => {
       .reduce(
         (acc, searchParameter) =>
           searchParameter.getSearchParameters.call(
-            { context: searchParameter.context },
+            {
```

Comment:
Since the function is now bound to his instance we can call it directly without the `call`.